code/game/objects/items/tools/maintenance_tools.dm

```diff
@@ -511,6 +511,7 @@
 
 /obj/item/maintenance_jack/Initialize()
 	. = ..()
+	flags_atom |= QUICK_DRAWABLE
```

Comment:
why not just put this in the definition itself? Same can be argued for doing the same for `/obj/item/weapon` but i'm slightly less pushy on that one

Reply:
To not override other flags by accident